framework/src/application.ts

```diff
@@ -300,7 +300,7 @@ export class Application {
 		this.logger.debug({ dbFolder }, 'Create module.db database instance.');
 		this._moduleDB = new Database(path.join(dbFolder, 'module.db'));
 		this.logger.debug({ dbFolder }, 'Create state.db database instance.');
-		this._stateDB = new StateDB(path.join(dbFolder, 'state.db'));
+		this._stateDB = new StateDB(path.join(dbFolder, STATE_DB_NAME));
```

Comment:
IMO, it's a good practice that you introduced `STATE_DB_NAME` constant.

1. Since now we have a constant, wouldn't it be good to replace all other occurrences of `'state.db'` string in the codebase with `STATE_DB_NAME` constant [including the debug message on the line above]?
2. How about doing the same for `'module.db'` string?